### ISSUE TITLE
gpg's dirmngr crashes with a segmentation fault

### ISSUE THREAD

**Issue by dscho** [git-for-windows/git#3771]
I'll just skip the setup details because this bug was most certainly introduced with -rc0, where this happens:

```console
$ dirmngr --help
Segmentation fault (core dumped)
```

I'm on it.

**Comment by dscho** [comment]
```
3a2603685bf49b37b66e60e6ef931315798bdb15 is the first bad commit
commit 3a2603685bf49b37b66e60e6ef931315798bdb15
Author: Git for Windows Build Agent <ci@git-for-windows.build>
Date:   Tue Mar 22 03:20:08 2022 +0000

    Update 1 package

    libgnutls (3.7.1-1 -> 3.7.4-1)
```

**Comment by dscho** [comment]
Let's see if [building a new `gnupg` package](https://dev.azure.com/Git-for-Windows/git/_build/results?buildId=97339&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a) helps.

**Comment by dscho** [comment]
Fixed by [git-for-windows/MSYS2-packages@382d10b06](https://github.com/git-for-windows/MSYS2-packages/commit/382d10b06bdb05bf422efed4db4ee53e98f9228c#diff-17338db931b293494070fb155fc308e796e290b0bbc6c17d91c664517dac8deb) (via [this build](https://dev.azure.com/Git-for-Windows/git/_build/results?buildId=97384&view=logs&j=0b109914-fd3b-548c-e58c-f9d47cb04f5f&t=a34f0d9a-4cf1-52c6-8b55-ed5de40333f4)).
